mapit_gb/management/commands/mapit_UK_add_slugs_to_local_authorities.py

```diff
@@ -0,0 +1,38 @@
+# This script is for a one off import of govuk slugs for local authorities
+
+import json
+from django.core.management.base import NoArgsCommand
+from mapit.models import Area, CodeType
+from django.utils.six.moves import urllib
+
+
+class Command(NoArgsCommand):
+    help = 'Assigns slugs from frontend to local authorities'
+
+    def handle_noargs(self, **options):
+        try:
+            code_type = CodeType.objects.get(code='govuk_slug')
+        except CodeType.DoesNotExist:
+            code_type = CodeType(
+                code='govuk_slug',
+                description='Slug for use by GOV.UK'
+            )
+            code_type.save()
+
+        url = "https://raw.githubusercontent.com/alphagov/frontend/master/lib/data/authorities.json"
```

Comment:
We're putting slugs into MapIt so that they're available centrally and consistently to all apps, which means that MapIt should become the canonical source of LA slugs. Once we've got the slugs in MapIt we want to delete this file from Frontend and use the slugs from MapIt instead there, and also use the slugs from MapIt in local links manager and delete the import which uses this file. Instead of importing the slugs into MapIt from this file in Frontend we should store the data in this repo and load it from here.